core/src/main/scala/org/apache/spark/util/collection/ExternalSorter.scala

```diff
@@ -368,8 +368,7 @@ private[spark] class ExternalSorter[K, V, C](
     val bufferedIters = iterators.filter(_.hasNext).map(_.buffered)
     type Iter = BufferedIterator[Product2[K, C]]
     val heap = new mutable.PriorityQueue[Iter]()(new Ordering[Iter] {
-      // Use the reverse of comparator.compare because PriorityQueue dequeues the max
```

Comment:
shall we keep this comment? we still use reverse order here.

Reply:
Maybe: `Use the reverse order because PriorityQueue dequeues the max`.